apps/evm/src/constants/chainMetadata.ts

```diff
@@ -97,6 +98,14 @@ export const CHAIN_METADATA: {
     lstPoolVWstEthContractAddress: '0x253515E19e8b888a4CA5a0a3363B712402ce4046',
     nativeToken: getToken({ chainId: ChainId.ARBITRUM_SEPOLIA, symbol: 'ETH' })!,
   },
+  [ChainId.ZKSYNC_MAINNET]: {
+    name: 'zkSync',
+    logoSrc: zkSyncLogo,
+    explorerUrl: 'https://explorer.zksync.io/',
```

Comment:
I think this URL should not include the last `/`. It works, but the built URL has two slashes